contribute/modules/ROOT/pages/vtkjs.adoc

```diff
@@ -9,7 +9,7 @@ In the examples and benchmarks, we generate 2D and 3D dataset that can be visual
 
 == Building a VTKJS file
 
-A {vtkjs} file can be produced using Paraview and a Python script from {feelpp} than can be injected into the Paraview interface.
+A {vtkjs} file can be produced using Paraview and a Python script from {feelpp} than can be injected into the Paraview interface. The Python script can be downloaded https://github.com/Kitware/vtk-js/blob/master/Utilities/ParaView/export-scene-macro.py[here]. This script writes the vtkjs file in `$HOME/vtkJsExport` on your computer.
```

Comment:
The python script can be downloaded from here 
https://github.com/feelpp/feelpp/blob/develop/tools/scripts/ParaView/export-scene-macro.py